api/swagger.yml

```diff
@@ -539,6 +539,8 @@ components:
           type: string
         email:
           type: string
+        password:
+          type: string
```

Comment:
```suggestion

```
We don't want to return this information about the user

Reply:
True. Added it during debugging and forgot to remove it. Fixed